okhttp/src/jvmTest/java/okhttp3/internal/idn/IdnaMappingTable.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.internal.idn
+
+import okio.BufferedSink
+
+interface IdnaMappingTable {
+
+  /**
+   * Returns true if the [codePoint] was applied successfully. Returns false if it was disallowed.
+   */
+  fun apply(codePoint: Int, sink: BufferedSink): Boolean
```

Comment:
nit: `apply` not as an operator seems a bit `doStuff`

Reply:
Good call. Renamed to `map`